index.html

```diff
@@ -492,6 +513,27 @@ <h3 class="card__title">Comments</h3>
         document.querySelector("input[name=siteUrl]").value = addProtocol(siteUrl);
       };
 
+      const toggleFormStatus = (e) => {
+        const isBrandedContentChecked = e.srcElement.classList.contains('branded-content-card');
+
+        if(isBrandedContentChecked){
+          const siteUrlInput = buttonsToDisable[0];
+
+          myForm.classList.add('form--blocked');
+          siteUrlInput.value = 'https://playground.mrf.io/templates/branded-content-example.html'
+        } else {
+          myForm.classList.remove('form--blocked');
+        }
+
+        for (let i = 0; i < buttonsToDisable.length; i++) {
+          buttonsToDisable[i].disabled = isBrandedContentChecked;
+        }
+      };
+
+      for (let i = 0; i < cards.length; i++) {
+        cards[i].addEventListener("change", toggleFormStatus);;
+      }
```

Comment:
It's not wrong but, since we're already using other ES6 features, maybe we can use `[...document.querySelectorAll(".whatever")].forEach` here.